app/web/features/auth/locales/en.json

```diff
@@ -16,11 +16,13 @@
   },
   "change_password_form": {
     "title": "Change password",
+    "subtitle": "Please change password",
```

Comment:
"Use this form to reset your login password."

Reply:
implemented ✅